Pinta.Effects/Classes/ErrorDiffusionMatrix.cs

```diff
@@ -0,0 +1,140 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+
+namespace Pinta.Effects;
+
+public enum PredefinedDiffusionMatrices
+{
+	Sierra,
```

Comment:
These should probably get Caption attributes added, e.g. `Floyd-Steinberg` should have a hyphen in it from what I see online, along with explanations for translators

It would also be good to see if there are any more descriptive names for users, e.g. "Fake" Floyd-Steinberg might not help a user understand what it does

Reply:
Sure. I added a `CaptionAttribute` to those values that have hyphens or spaces; and I labeled the matrix you mentioned as "Floyd-Steinberg Lite"

Reply:
Thanks - since these captions will also show up as translatable strings, also adding some `Translators: ...` comments would be good to explain to translators that these are algorithms named after people

Reply:
Sure. I just added comments for translators making it clear that the matrices were named after people